src/index.js

```diff
@@ -571,10 +573,18 @@ function postgresToProperty(metadata) {
   var property = {name: metadata.column_name};
   switch (metadata.data_type) {
     case 'integer':
+    case 'boolean':
     case 'text':
     case 'date':
       property.type = metadata.data_type;
       break;
+    case 'bigint':
+      property.type = 'integer';
+      break;
+    case 'real':
+    case 'double precision':
+      property.type = 'numeric';
```

Comment:
property.type should contain a valid json schema type then this line better be `property.type = 'number'`